docs/how-to-guides/metabox.md

```diff
@@ -243,7 +243,7 @@ When the meta box area is saving, we display an updating overlay, to prevent use
 
 An example save url would look like:
 
-`mysite.com/wp-admin/post.php?post=1&action=edit&meta-box-loader=1`
+`example.com/wp-admin/post.php?post=1&action=edit&meta-box-loader=1`
```

Comment:
Very weakly-held opinion - would `example.org` reflect more the non-profit nature of the WordPress org?